doc/integration-staging-sync.md

```diff
@@ -0,0 +1,41 @@
+# Integration & Staging Sync
+
+## Overview
+
+Every morning the data in our integration and staging databases is synchronised
+with production. We also need to keep GovDelivery in sync so that the
+GovDelivery Topic IDs that we hold in our database continue to link to the
+correct topics in every environment.
+
+This is complicated by the fact that both integration and staging talk to the
+same staging 'instance' of GovDelivery. Despite being separate accounts, the
+GovDelivery topic ids have to be unique across the entire instance.
+
+To achieve the synchronisation we have a rake task
+(`sync_govdelivery_topic_mappings`) which is automatically triggered when the
+'Data Sync Complete' job runs in Jenkins.
```

Comment:
Can you link to https://deploy.integration.publishing.service.gov.uk/job/Data_Sync_Complete/?